Tema-5/HW4/style.css

```diff
@@ -0,0 +1,12 @@
+*{
+    box-sizing: border-box;
+    width: 60%;
```

Comment:
Nu e recomandat sa pui width in universal selectors.

Reply:
Ok, am modificat si am scos selectorii universali